libtsuba/test/CMakeLists.txt

```diff
@@ -1,22 +1,38 @@
-add_executable(manifest-test manifest.cpp)
-target_link_libraries(manifest-test tsuba)
-target_include_directories(manifest-test PRIVATE ../src)
-add_test(NAME manifest COMMAND manifest-test ${BASEINPUT}/propertygraphs/rmat15/katana_vers00000000000000000001_rdg.manifest)
-set_property(TEST manifest APPEND PROPERTY LABELS quick)
-
-add_executable(file-view-test file-view.cpp)
-target_link_libraries(file-view-test tsuba)
-add_test(NAME file-view COMMAND file-view-test "${CMAKE_CURRENT_BINARY_DIR}/file-view-test-wd")
-set_tests_properties(file-view PROPERTIES FIXTURES_REQUIRED file-view-ready LABELS quick)
-add_test(NAME clean-file-view COMMAND ${CMAKE_COMMAND} -E rm -rf "${CMAKE_CURRENT_BINARY_DIR}/file-view-test-wd")
-set_tests_properties(clean-file-view PROPERTIES FIXTURES_SETUP file-view-ready LABELS quick)
-
-add_executable(parquet-test parquet.cpp)
-target_link_libraries(parquet-test tsuba)
-add_test(NAME parquet COMMAND parquet-test "${CMAKE_CURRENT_BINARY_DIR}/parquet-test-wd")
-set_tests_properties(parquet PROPERTIES FIXTURES_REQUIRED parquet-ready LABELS quick)
-add_test(NAME clean-parquet COMMAND ${CMAKE_COMMAND} -E rm -rf "${CMAKE_CURRENT_BINARY_DIR}/parquet-test-wd")
-set_tests_properties(clean-parquet PROPERTIES FIXTURES_SETUP parquet-ready LABELS quick)
+set(name manifest)
+set(test_name ${name}-test)
+add_executable(${test_name} manifest.cpp)
+target_link_libraries(${test_name} tsuba)
+target_include_directories(${test_name} PRIVATE ../src)
+if(KATANA_USE_INPUTS)
+  add_dependencies(${test_name} input)
+endif()
```

Comment:
I'm a little unclear on who sets KATANA_USE_INPUTS and why it needs this check needs to be propagated to so many places.  Doesn't this test specify an input from ${BASEINPUT}/propertygraphs?  Doesn't that mean it is using inputs?

Reply:
So previously if you built just `libtsuba` or just `libgalois`, you wouldn't build `input` as well and your tests would fail. So I wanted to add `input` as a dependency, but in some CI cases `KATANA_USE_INPUTS` is false and the inputs get provided in a different way. So we have to do this.